spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierResolver.java

```diff
@@ -51,37 +51,30 @@ public class TypeQualifierResolver {
     static final ClassDescriptor elementTypeDescriptor = DescriptorFactory
             .createClassDescriptor(ElementType.class);
 
-    static final ClassDescriptor androidNullable = DescriptorFactory.createClassDescriptor("android/support/annotation/Nullable");
```

Comment:
Define constants for the class names to avoid repetition.

`private static final String ANDROID_SUPPORT_ANNOTATION = "android/support/annotation/";`